test/frontend_test.dart

```diff
@@ -0,0 +1,457 @@
+            .findElements(const By.xpath(
+                '*//div[contains(@class, "menu-item--services")]/div[contains('
+                    '@class, "nav-menu")]/div[contains(@class, "menu-item")]'))
+            .length;
+        await waitForAngular(driver);
+        expect(await isXAtLeastValueOfY(menuItemsLength, 8), true);
+      });
+
+      test('szolgaltatasaink menu has trial after login', () async {
+        var trialPeriodItem = await driver.findElements(const By.xpath(
+            '*//div[contains(@class, "nav-menu")]/div[contains(@class, "menu-item")]'
+            '/a[contains(@href,"/profile/product/probaidoszak")]'));
+        expect(futureElementExists(trialPeriodItem), true);
+      });
+
+      test('szolgaltatasaink menu has settings after login', () async {
+        var settingsItem = await driver.findElements(const By.xpath(
+            '*//div[contains(@class, "nav-menu")]/div[contains(@class, "menu-item")]'
+            '/a[contains(@href,"/profile/settings")]'));
+        expect(futureElementExists(settingsItem), true);
+      });
+
+      test('szolgaltatasaink menu has logout after login', () async {
+        var logoutItem = await driver.findElements(const By.xpath(
+            '*//div[contains(@class, "nav-menu")]/div[contains(@class, "menu-item")]'
+            '/a/span/font/font[contains(text(),"egress")]'));
+        expect(futureElementExists(logoutItem), true);
+      });
+    });
+
+    group("probaidoszak page tests |", () {
+      setUpAll(() async {
+        await waitForAngular(driver);
+        await loginUserWithCredentials(
+            newUserEmail, allUserPassword, driver, true);
+        await navigateToPage(driver, 'profile/product/probaidoszak');
+        await waitForAngular(driver);
+        await sleep5();
+      });
+
+      test('you see a bunch of videos showing up', () async {
+        var videoItemsLength = await driver
+            .findElements(const By.cssSelector(
+                'video-summary-list > div.training-type-list > div '
+                '> video-summary > div.video-summary'))
+            .length;
+        print("Video items found: " + videoItemsLength.toString());
+        expect(await isXAtLeastValueOfY(videoItemsLength, 5), true);
+      });
+
+      test('video popup shows on page after clicking a video', () async {
+        var firstVideoItem;
+        firstVideoItem = await driver.findElement(const By.xpath(
+            '*//video-summary-list/div[contains(@class, '
+                '"training-type-list")]//video-summary[1]'));
+        await firstVideoItem.click();
+        await sleep5();
+        var videoPlayer = await driver.findElement(const By.xpath(
+            '//*[@id="default-acx-overlay-container"]//vimeo-player/'
+            'div[contains(@class,"modal-body")]/div/div/iframe'));
+        await videoPlayer.click();
+        await sleep2();
+        await waitForAngular(driver);
+        expect(await elementExists(videoPlayer), true);
```

Comment:
You already clicked on this element three lines above - so it's not likely that it does not exists here.